tests/integration/index.test.ts

```diff
@@ -0,0 +1,74 @@
+import { ANT, AOProcess, ArweaveSigner, createAoSigner } from '@ar.io/sdk';
+import { strict as assert } from 'node:assert';
+import * as fs from 'node:fs';
+import { describe, it } from 'node:test';
+
+import {
+  AO_LOADER_HANDLER_ENV,
+  AO_LOADER_OPTIONS,
+  LocalAO,
+  TEST_AOS_ANT_WASM,
+} from './utils.js';
+
+const testWalletJSON = fs.readFileSync('./test-wallet.json', {
+  encoding: 'utf-8',
+});
+
+const testWallet = JSON.parse(testWalletJSON);
+const signers = [
+  new ArweaveSigner(testWallet),
+  createAoSigner(new ArweaveSigner(testWallet)),
+] as const;
+
+describe('integration esm tests', async () => {
+  async function createLocalANT() {
```

Comment:
is it possible to make this `createLocalProcess` and specify which wasm to use?

Reply:
[7b64273](https://github.com/ar-io/ar-io-sdk/pull/387/commits/7b64273c070d95bb14e372f8ee1e813b09180793)